lib/clipboard.zsh

```diff
@@ -30,6 +30,17 @@ function clipcopy() {
     else
       cat $file > /dev/clipboard
     fi
+  elif [[ $OSTYPE == linux-android ]]; then
```

Comment:
In my device, `$OSTYPE` shows `linux-androideabi`. Using `linux-android*` instead will also match this.